content/outcomes/secure-software-supply-chain/establish-capability/establish-devsecops.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Developing a DevSecOps Practice
+weight: 1
+layout: single
+team:
+- Craig Tracey
+---
+
+Modern software delivery demands that we think differently about how we bring
+code to production. Traditional methodologies typically took on a siloed approach:
+a development team writes the code, a security team ensures that this code
+adheres to organizational compliance requirements, and operations teams
+maintained the availability of the developed service. While this organization of
+specific roles often met the demand for bringing code to production, it suffers
+from a number of drawbacks.
+
+First, it was often slow. With different groups all involved in bringing code to
+production, responsibilities were often serialized. Once the code is ready, the
+security team takes a look. And then, once the security team blesses the
+release, an operator can deploy and maintain it. Yes, these teams could
+multi-task, and begin the next body of work while their current release
+candidate was being validated. But, this is typically not an efficient process.
+In practical terms, it introduces disruptive context switching, but it also
+means that these teams are not working collaboratively towards a common goal.
+
+Because these teams each have different objectives, friction often develops when
```

Comment:
Different objectives *and* different incentives.

Development teams are incentivized to go fast and release as many new features as quickly as possible, while security and operations teams are the most risk averse and are incentivized to keep things running the way they are while introducing as little change as possible.